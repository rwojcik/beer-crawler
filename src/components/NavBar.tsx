import AppBar from "@material-ui/core/AppBar";
import Toolbar from "@material-ui/core/Toolbar";
import Typography from "@material-ui/core/Typography";
import React from "react";

export const NavBar = () => {
  return (
    <AppBar position="static">
      <Toolbar>
        <Typography variant="title" color="inherit">
          BeerCrawler
        </Typography>
      </Toolbar>
    </AppBar>
  );
};
