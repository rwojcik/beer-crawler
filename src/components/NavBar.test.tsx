import React from "react";
import ReactDOM from "react-dom";
import { NavBar } from "./NavBar";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<NavBar />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("renders title", () => {
  const div = document.createElement("div");
  ReactDOM.render(<NavBar />, div);
  expect(div.textContent).toContain("BeerCrawler");
  ReactDOM.unmountComponentAtNode(div);
});
