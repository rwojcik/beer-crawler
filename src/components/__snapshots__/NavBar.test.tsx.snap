// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <header
    class="MuiPaper-root-162 MuiPaper-elevation4-168 MuiAppBar-root-153 MuiAppBar-positionStatic-157 MuiAppBar-colorPrimary-160"
  >
    <div
      class="MuiToolbar-root-189 MuiToolbar-regular-191 MuiToolbar-gutters-190"
    >
      <h2
        class="MuiTypography-root-193 MuiTypography-title-199 MuiTypography-colorInherit-222"
      >
        BeerCrawler
      </h2>
    </div>
  </header>
</div>
`;
